packages/create-app/src/index.ts

```diff
@@ -29,8 +29,11 @@ const main = (argv: string[]) => {
   program
     .name('backstage-create-app')
     .version(version)
-    .description('Creates a new app in a new directory or specified [path]')
-    .usage('[path] [options]')
+    .description('Creates a new app in a new directory or specified path')
+    .option(
```

Comment:
@benjdlambert , I've updated the code to use the `--path` option in favor of the `[path]` argument. Let me know what you think; I'm happy to tweak things until we're happy with the UX.